<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BarCodeScaleWidget</class>
 <widget class="QWidget" name="BarCodeScaleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="BCSWidget" name="bcsPage">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#bcsPage{
	background-color: rgb(81, 80, 80);
}
</string>
      </property>
     </widget>
     <widget class="BCSGoodsSelectWidget" name="bcsGoodsSelectPage">
      <property name="styleSheet">
       <string notr="true">QWidget#bcsGoodsSelectPage{
	background-color: rgb(81, 80, 80);
}</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BCSGoodsSelectWidget</class>
   <extends>QWidget</extends>
   <header>BCSGoodsSelectWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BCSWidget</class>
   <extends>QWidget</extends>
   <header>BCSWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
